packages/components/src/templates/next/components/complex/ContactInformation/components/filterContactMethods.ts

```diff
@@ -9,12 +9,20 @@ export const filterContactMethods = ({
   methods,
   whitelistedMethods,
 }: FilterContactMethodsProps) => {
+  const nonEmptyMethods = methods.filter(
+    (method) =>
+      Array.isArray(method.values) &&
+      method.values.filter(
+        (value) => typeof value === "string" && value.trim() !== "",
+      ).length > 0,
+  )
```

Comment:
question: should we also remove the values where `value.trim()` is `""`

Reply:
eh no at this point it just filters for an array with non-empty values (ie, `["            ", "", "hello"]`) 

but i'm suggesting to remove the empty values so that it's (`["hello"`])